### PR TITLE
Graphql: Remove cityId from RouteStop type

### PR DESCRIPTION
**Pull request by tbergq**
Rename airport to stop, since it could be any kind of stop

see discussion in #322

### CODE REVIEW

**Comment by FilipMessa** [comment]
@michalsanger can you please look on this PR also? thanks

File: apps/graphql/src/dataloaders/itinerariesHelpers.js [hunk 1]
```diff
@@ -151,17 +151,17 @@ const apiRouteItemToDeparture = (routeItem: ?ApiRouteItem) => ({
...
   time,
 }: {|
   +cityName: ?string,
-  +cityId: ?string,
+  +code: ?string,
   +time: {| +utc: ?string, +local: ?string |},
 |}): RouteStop => ({
   cityName,
-  cityId,
+  cityId: null,
```
Comment (michalsanger) [inline apps/graphql/src/dataloaders/itinerariesHelpers.js]: Why is `cityId` not removed from the `RouteStop`?

File: apps/graphql/src/types/output/RouteStop.js [hunk 2]
```diff
@@ -9,18 +9,19 @@ import type { GraphqlContextType } from '../../services/GraphQLContext';
 
 export default new GraphQLObjectType({
   name: 'RouteStop',
-  description: 'Departure or arrival for a segment',
+  description: 'Departure or arrival for a segment or sector',
   fields: {
     cityName: {
```
Comment (michalsanger) [inline apps/graphql/src/types/output/RouteStop.js]: I suggest to remove also `cityName`. The `Location` entity should have new property `city: LocationArea` from where name of the city can be resolved as `stop?.city?.name`

File: apps/graphql/src/dataloaders/itinerariesHelpers.js (hunk 1 above)
Comment (tbergq) [reply apps/graphql/src/dataloaders/itinerariesHelpers.js]: Because tequila does not return the cityId, it returns the iata code of the stop.

Comment (tbergq) [reply apps/graphql/src/dataloaders/itinerariesHelpers.js]: We have cityId from the mocked data from MMB, though we are currently not using it, so I guess I can remove it, and add it later if we need it

File: apps/graphql/src/types/output/RouteStop.js (hunk 2 above)
Comment (tbergq) [reply apps/graphql/src/types/output/RouteStop.js]: Yes, that should probably be doable. I see the location query returns a field called city, we should be bale to get it from there.

File: apps/graphql/src/dataloaders/itinerariesHelpers.js (hunk 1 above)
Comment (tbergq) [reply apps/graphql/src/dataloaders/itinerariesHelpers.js]: Sorry, scratch that, I use it to get `destinationImageUrl` in `BookingInterface`. So cityId has to stay in the resolvers.

**Comment by tbergq** [comment]
@michalsanger 🚀

**Pull request merged**